.buildkite/cache-builder.yml

```diff
@@ -0,0 +1,23 @@
+# This pipeline runs via Buildkite's scheduled jobs feature.
+#
+# It's meant to rebuild various CI caches on a periodic async basis, so as
+# not to waste time on every CI job updating the cache.
+
+common_params:
+  - &common_plugins
+    - automattic/a8c-ci-toolkit#2.18.2
+    - automattic/git-s3-cache#1.1.4:
+        bucket: a8c-repo-mirrors
+        repo: automattic/gutenber-mobile/
+
+steps:
+  # Build the Git Repo cache
+  #
+  # Because this repo is so large, we periodically create a Git Mirror and copy it to S3,
+  # from where it can be fetched by agents more quickly than from GitHub, and so that
+  # agents then have less new commits to `git pull` by using that cache as starting point
+  - label: ':git: Rebuild git cache'
+    command: cache_repo a8c-repo-mirrors
+    plugins: *common_plugins
+    agents:
+      queue: default
```

Comment:
Other repos cache the pods as well. I skipped this here because the command used for the caching expects `Podfile` to be in the root.

We'll get a good speed bump from caching the Git repo. We can consider caching the pods later on.